src/main/com/example012/v1/model/StringConstant.java

```diff
@@ -0,0 +1,16 @@
+package main.com.example012.v1.model;
+
+public enum StringConstant {
```

Comment:
Celkově už je to mnohem lepší. 

1 - apka by se neměla asi spoustet tim, ze vytvoris LotteryGameController ... ale mel by jsi s nim neco udelat...neco jako 
lottery.play() 
ten konstruktor to jen vytvori s pozdaovanymi daty, nic moc navic by nemel delat 

2 - tato classa byla na procviceni enumů, ale uz tam jeden mame, ... tak tady ten enum zmen na classu a ty promenne zmen na

public static final String XXX = "xxx";     ... pak k tomu jen pristoupis StringConstants.XXX ... nemusi se volat value ... enum je pro tento pripad trosku overkill, ale pouziva se to tak ... lepe se enum hodi pro ty nase Lottery ;-) 

3 - pofixovat vse zlute highlighted od idey a muzeme zavrit ... jdou tam drbky, ale je treba se posunout

Reply:
1, Upraveno, teď pouze inicializace hodnot a logika se bude odehrávat až právě v metodě play 
2, Změněno a upraveno view 
3, Hotovo, ve všech souborech je aktuálně 0 varování

Reply:
Mi svítí ještě toto ... jeslti pak víš proč?

![image](https://github.com/Silluh/Learning/assets/5745308/2210cfa0-3688-4261-be79-5891413f73a4)

Reply:
velké "W", přejmenuji